pkg/operator/options/suite_test.go

```diff
@@ -54,7 +54,7 @@ var _ = Describe("Options", func() {
 		"KUBE_CLIENT_QPS",
```

Comment:
I see the feature gate for drift is dropped in the other PR. Just sanity checking: Is this also already dropped from the feature gate parsing logic that we have here?

Reply:
Yes addressed in https://github.com/kubernetes-sigs/karpenter/commit/2129147cb33512855ed672acaeedbce9f45ec0af#diff-77e3ca1abc21984035279a7b6ebc696e5c58f35dcc7386ea4bb3333c332ab917